database.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/sqlite"
+)
+
+type staticCommand struct {
+	ID        uint `gorm:"primary_key"`
+	CreatedAt time.Time
+	CreatedBy string
+	UpdatedAt time.Time
+	UpdatedBy string
+	Call      string
+	Message   string
+	LastUseBy string
+	LastUse   time.Time
+}
+
+func (b *bot) newDatabase() {
+	var err error
+	b.db, err = gorm.Open("sqlite3", b.config.Database)
+	if err != nil {
+		log.Fatalf("failed opening database: %s", err)
+	}
+	b.db.AutoMigrate(&staticCommand{})
```

Comment:
gorm has a verbose logging function, probably use that and dumb to the debuglogger or look for gorm errors. Most functions have an `.Error` member iirc.